src/main.cpp

```diff
@@ -235,11 +241,31 @@ int main(int argc, char **argv) {
 		return EXIT_FAILURE;
 	}
 
+   PlayerInputComponent* playerInputComp = new PlayerInputComponent();

```

Comment:
Let's leave it here for now. I want to map out a more robust Camera hierarchy. I haven't decided how exactly I want to do it yet.